packages/docs/package.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "@blueprintjs/core": "^1.7.0",
     "@blueprintjs/datetime": "^1.6.0",
-    "@blueprintjs/table": "^1.4.0",
+    "@blueprintjs/table": "^1.4.1",
```

Comment:
awkward to bump dep without changing package version but it's private so whatever